Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceSerializer.cpp

```diff
@@ -23,6 +23,53 @@ namespace AzToolsFramework
     {
         AZ_CLASS_ALLOCATOR_IMPL(JsonInstanceSerializer, AZ::SystemAllocator, 0);
 
+        namespace Internal
+        {
+            inline static const char* PathStartingWithEntities = "/Entities/";
+            inline static const char* PathMatchingContainerEntity = "/ContainerEntity";
+
+            //! Identifies the instance members to reload by parsing through the patches provided.
+            static void IdentifyInstanceMembersToReload(
+                PrefabDom& patches, AZStd::unordered_set<EntityAlias>& entitiesToReload, bool& shouldReloadContainerEntity)
+            {
+                for (const PrefabDomValue& patchEntry : patches.GetArray())
+                {
+                    PrefabDomValue::ConstMemberIterator patchEntryIterator = patchEntry.FindMember("path");
+                    if (patchEntryIterator != patchEntry.MemberEnd())
+                    {
+                        AZStd::string patchPath = patchEntryIterator->value.GetString();
+                        if (patchPath.starts_with(PathStartingWithEntities))
+                        {
+                            patchPath.erase(0, strlen(PathStartingWithEntities));
+                            AZStd::size_t pathSepartorIndex = patchPath.find('/');
```

Comment:
typo: pathSeparatorIndex -> pathSeparatorIndex
```suggestion
                            AZStd::size_t pathSeparatorIndex = patchPath.find('/');
```